test/test.org.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var Github = require('../src/github.js');
+var testUser = require('./user.json');
+var github, organization;
+
+describe('Github.Organization', function() {
+   before(function() {
+      github = new Github({
+         username: testUser.USERNAME,
+         password: testUser.PASSWORD,
+         auth: 'basic'
+      });
+      organization = github.getOrg();
+   });
+
+   it('should create an organisation repo', function(done) {
+      var repoTest = Math.floor(Math.random() * (100000 - 0)) + 0;
+      var options ={
+          "orgname": testUser.ORGANIZATION,
```

Comment:
Properties of an object, unless they are an invalid identifier, don't need to be in quotes. Can you remove them?